File: json/narwhal/read/json_read.js
#! /usr/local/bin/narwhal
// ------------------------------------------------------------------
//
//	json_read.js
//
//						May/07/2011
//
// ------------------------------------------------------------------
var text_manipulate = require ('/var/www/data_base/common/narwhal_common/text_manipulate.js');
// ------------------------------------------------------------------

print	("*** 開始 ***");

var args = require('system').args;
var file_in = args[1];

var json_str = readFile(file_in);

var dict_aa = JSON.parse (json_str);

text_manipulate.dict_display_proc (dict_aa);

print	("*** 終了 ***");
// ------------------------------------------------------------------

File: json/jython/read/json_read.py
#! /opt/jython/bin/jython
# -*- coding: utf-8 -*-
#
#	json_read.py
#
#					Oct/12/2016
#
#
import	sys
import	json
#
reload(sys)
sys.setdefaultencoding("utf-8")
#
sys.path.append ('/var/www/data_base/common/python_common')
sys.path.append ('/var/www/data_base/common/jython_common')
from jython_file_io import file_to_str_proc
from text_manipulate import dict_display_proc
# -------------------------------------------------------------
sys.stderr.write ("*** 開始 ***\n")
#
file_in = sys.argv[1]
#
json_str = file_to_str_proc (file_in)
#
dict_aa = json.loads (json_str)
dict_display_proc (dict_aa)
#
sys.stderr.write ("*** 終了 ***\n")
# -------------------------------------------------------------
